README.md

```diff
@@ -18,8 +18,8 @@ The binaries for the bundle, bundled DEB and ActivityWatch watchers replacement
 - Run `sudo unzip aw-awatcher.zip -d /usr/local/bin` in the console to allow ActivityWatch to detect its presence.
   - Or install the provided **aw-awatcher_\*.deb**.
 - Remove `aw-watcher-window` and `aw-watcher-afk` from autostart at `aw-qt/aw-qt.toml` in [config directory](https://docs.activitywatch.net/en/latest/directories.html#config),
-  add `aw-awatcher`
-- Restart ActivityWatch. In the Modules submenu there should be a new checked module **aw-awatcher**.
+  add `aw-awatcher`.
+- Restart ActivityWatch. In the Modules submenu there should be a new checked module **aw-awatcher**. Note that awatcher shows up in the Web UI under Timeline as `aw-watcher-windows_$HOSTNAME`.
```

Comment:
```suggestion
- Restart ActivityWatch. In the Modules submenu there should be a new checked module **aw-awatcher**. Note that awatcher shows up in the Web UI under Timeline as `aw-watcher-window_$HOSTNAME`.
```
By the way, did you want to clarify that it's displayed with the same identifier as the builtin watcher watcher? Because if it's not, it would be a bug since it should be the same.

Reply:
It is needed to keep the correct statistics in dashboards since this just replaces the bundled watchers.

Reply:
It is displayed with the same name, I just wanted to clarify (not a bug). Yes that's what I expected, it was just unexpected.